### PR TITLE
Add half-page movements

### PR DESCRIPTION
**Pull request by elvis-sik**
Closes issue #215

### CODE REVIEW

File: command_mode.c [hunk 1]
```diff
@@ -1763,6 +1763,20 @@ static void cmd_win_pg_up(char *arg)
 	editable_unlock();
 }
 
+static void cmd_win_hf_pg_down(char *arg)
+{
+	editable_lock();
+  window_half_page_down(current_win());
```
Comment (flyingmutant) [inline command_mode.c]: Indentation is off here.

**Comment by flyingmutant** [comment]
Can you also add the documentation for new commands?

**Comment by elvis-sik** [comment]
I should have created a branch. Anyway, I did it. Any more problems?

**Comment by flyingmutant** [comment]
Merged as 82c0f1faebbdfd664e2a90d3c2c9422a93a63a1f, thanks!

**Comment by elvis-sik** [comment]
No, thank you for this awesome player! :smile:

**Pull request closed without merging**